webrtc.html

```diff
@@ -517,6 +517,10 @@
 
         <ol>
           <li>
+            <p>Let <var>p</var> be a new promise</p>.
+          </li>
+
+          <li>
             <p>Append an object representing the current call being handled
```

Comment:
Of course objects can be executed:  obj.execute().  That's what I had in one of my previous versions of the proof-of-concept code.  The fact that it's a function is an implementation detail.